### DISCUSSION TITLE
Why is the structurize different on custom forge profiles

### DISCUSSION THREAD

**Discussion by Somethingsfishyc** [ldtteam/minecolonies#8003] [Q&A]
I'm playing in 1.18.1 minecolonies but with my own custom profile on Forge but i've noticed that it shows every thing that i can build without having main item the item. But on the Official Minecolonies mod pack I had to have the main item to build the building. Is there a way i can configure my version of it the same way as the Minecolonies mod pack?

**Comment by uecasm** [comment]
For support questions, please use the Discord instead.
